modules/app-contentstudio/src/main/resources/assets/js/app/remove/ContentDeleteDialog.ts

```diff
@@ -84,6 +85,8 @@ export class ContentDeleteDialog
                 this.messageId = api.notify.showWarning(
                     i18n('dialog.delete.dependency.warning'), false);
 
+                this.addClickIgnoredElement(NotifyManager.get().getNotification(this.messageId));
+
```

Comment:
We should handle the case when ``getNotification`` returns ``null`` and make sure that ``addClickIgnoredElement`` is either not called or not doing anything in this case.